components/dex.libsonnet

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
not blocking this PR, but we should make sure that the openshift identity provider works here out of the box as well, even without dex.. overall the message should be that dex is optional, but a good option

Reply:
super good point. I would rlly like to have integration tests for this as well, but we don't currently have a good option for kind-style openshift clusters. however, it should really be enough knowing that the openshift auth provider implements oidc